serving/deploying-with-private-registry.md

```diff
@@ -0,0 +1,143 @@
+# Deploying to Knative using a private container registry
+This guide walks you through deploying an application to Knative from source code in a git repository using a private container registry for the container image. The source code should contain a dockerfile. For this guide, we'll use this [simple app](https://github.com/mchmarny/simple-app), but you could use your own.
+
+
+## Set up a private container registry and obtain credentials.
+If you do not want your container image to be publicly available, you may want to use a private container registry. In this example, we'll use IBM Container Registry, but most of these concepts will be similar for other clouds.
+
+1. Ensure you have the [IBM Cloud CLI](https://cloud.ibm.com/docs/cli/reference/ibmcloud/download_cli.html#install_use) installed.  
+
+1. Install the container registry plugin:
+
+    ```
+    ibmcloud plugin install container-registry
+    ```
+
+1. Choose a name for your first namespace, and then create it. A namespace represents the spot within a registry that holds your images. You can set up multiple namespaces as well as control access to your namespaces by using IAM policies.
+
+    ```
+    ibmcloud cr namespace-add <my_namespace>
```

Comment:
Move command up so that it immediately follows the instruction, "Choose a name for your first namespace, and then create it." 

I realized this happens in a few places; I'll mark them, but not explain as thoroughly as I did here and the comment further down.

Reply:
Use a semicolon to link your instructional sentence to the command. (Also applies more broadly.)

Reply:
Thanks, updated.